packages/editor/src/functions/addMediaNode.ts

```diff
@@ -208,6 +208,7 @@ export async function addMediaNode(
     )
     return entityUUID
   } else if (url.includes('.uvol')) {
+    // TODO: detect whether to add LegacyVolumetricComponent or VolumetricComponent
```

Comment:
I don't believe we want to support the LegacyVolumetric in the future @CITIZENDOT